sources/main.cpp

```diff
@@ -93,6 +95,8 @@ main(int argc, char* argv[]) {
       LOG(error) << MESS(NetworkFileNotFound, runtimeConfig.networkFilePath) << LOG_ENDL;
       return EXIT_FAILURE;
     }
+    xmlInitParser();
+    xercesc::XMLPlatformUtils::Initialize();
```

Comment:
Doesn't that depend on a USE_XERCES kind of option on dynawo side ?

Reply:
this is required for all xml parsers beside IIDM